wire-ios-request-strategy/Sources/APIs/MessageAPI.swift

```diff
@@ -177,23 +167,23 @@ class MessageAPIV1: MessageAPIV0 {
         .v1
     }
 
-    override func broadcastProteusMessage(message: any ProteusMessage) async throws -> (Payload.MessageSendingStatus, ZMTransportResponse) {
+    override func broadcastProteusMessage(message encryptedMessage: Data,
+                                          expirationDate: Date?) async throws -> (Payload.MessageSendingStatus, ZMTransportResponse) {
```

Comment:
do we need expiration on broadcast messages

Reply:
@typfel why because if they fail of bad network, we don't display the error to the user ?

Reply:
It makes sense to have some timeout on the request, isn't there a default timeout?

Reply:
yes, the ones from URLSession so 30 sec without data transmission and 12min for the full request to complete